css/index.css

```diff
@@ -1,3 +1,347 @@
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+.container {
+	width: 880px;
+	margin: 0 auto;
+	display:flex;
+	flex-direction:column;
+}
+
+hr {
+  width: 100%;
+  color: black;
+}
+
+a:hover {
+	background: whitesmoke;
+	color: red;
+}
+
+a:link {
+	text-decoration: none;
+	color: grey;
+}
+
+a:visited {
+	color: grey;
+}
+
+.nav{
+	display: flex;
+	justify-content: space-between;
+	align-content: center;
+	padding: 10px 0px;
+  color: grey;
+}
+.nav a {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+.container .header  {
+	display: flex;
+	justify-content: space-between;
+	align-items: center; 
+	padding: 30px;
+}
+
+.container header section div p {
+	font-size: 20px;
+	text-align: center;
+	padding: 5px 40px;
+	border: 2px solid black;
+}
+
+.container .header-text {
+	text-align: center;
+	font-size: 90px;
+}
+
+.container .top-content {
+	display: flex;
+}
+
+.container .top-text {
+	padding: 10px;
+}
+
+.container .middle-img {
+	padding: 10px 0px;
+}
+
+.container .bottom-content {
+	display: flex;
+	padding: 10px 5px;
+}
+
+/* .container .bottom-text {
+} */
+
+.container .contact p {
+	padding: 5px 0px;
+}
+
+.container footer {
+	text-align:center;
+	padding: 20px 0px;
+}
+
+/* services.css */
+
+.SmallHR {
+	size: 2px;
+	padding: 2px;
+}
+
+.services-header-img {
+	border: 1px solid black;
+}
+
+.services-text {
+	display: flex;
+	flex-direction:column;
+	padding: 30px 0px;
+}
+
+.row {
+	display: flex;
+	justify-content: space-between;
+
+}
+
+.box {
+	border: 1px solid black;
+	background: #F4F3F4;
+	justify-content: space-between;
+	margin: 20px;
+	padding: 20px;
+}
+
+.LMbutton {
+	border: 1px black solid;
+	display: inline-block;
+	border-radius: 5px;
+	background: white;
+	margin: 20px 0px 0px 0px;
+	padding: 8px 30px;
+	font-size: 14px;
+}
+
+.services-facts {
+	display:flex;
+	padding: 20px 0px;
+}
+
+.services-facts-text {
+	margin: 0px 20px 0px 0px;
+}
+
+.fact-list {
+	padding: 20px;
+}
+
+.services-info-img {
+	border: 1px solid black;
+}
+
+.fact-list li {
+	padding: 5px;
+}
+
+
+
+/*---------------RESPONSIVE DESIGNING-----------*/
+
+@media (max-width: 800px) {
+  .container {
+    width: 100%;
+  }
+
+  .nav {
+    display: flex;
+    justify-content: space-between;
+    align-content: center;
+    padding: 10px 0px;
+    color: grey;
+  }
+
+  .nav a {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .nav a:last-child {
+    order: -1;
+  }
+
+  .logo-nav {
+    
+    width: 70%;
+  }
+
+  .header {
+    display: flex;
+    justify-content: center;
+    align-content: center;
+    border: 1px solid
```

Comment:
I'm guessing that this border was for debugging. Don't forget to remove things like this from your code.